README.md

```diff
@@ -43,7 +43,7 @@
 
 ## Stability, future changes, post 1.0 freeze
 
-Despite being at an early development stage, the V language is relatively stable, and doesn't
+Despite being at an early development stage, the V language is relatively stable and doesn't
```

Comment:
The comma is needed to separate the "thoughts".

Reply:
Done